charts/roman/templates/ingress.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: extensions/v1beta1
```

Comment:
please change the ingress apiVersion for maximum kubernetes version compatibility as done in https://github.com/wireapp/wire-server/pull/3002: add some function to _helpers.tpl and adjust the ingress.yaml in the same fashion as done in that PR for other charts, e.g. the nginx-ingress-services one.